### DISCUSSION TITLE
issue connecting kube-agent to teleport cluster

### DISCUSSION THREAD

**Discussion by TySmith17** [gravitational/teleport#6915] [Q&A]
Im having an issue connecting a kubernets cluster using the `teleport/teleport-kube-agent` chart to our cluster that is setup with the `teleport/teleport-cluster` chart. Both are running in GKE 

Error in the logs on the kube agent: 

```ERRO [PROC:1]    Kube failed to establish connection to cluster: Post "https://teleport.cluster.local/v2/tokens/register": dial tcp <IP_TELEPORT_CLUSTER_LB>:433: connect: connection timed out, Post "https://<PROXY_ADDRESS>:433/v1/webapi/host/credentials": dial tcp <IP_TELEPORT_CLUSTER_LB>:433: i/o timeout. service/connect.go:70 ```

We set up teleport cluster by following the getting started guide on https://goteleport.com/  and the kube agent by following the multiple clusters guide. Any sort of help would be greatly appreciated

**Comment by Valien** [comment]
Are you using port `433` for a specific reason? Should be `443`. Hey @stevenGravy - any ideas on this?
